webapp/app/assessment/fra/sections/extentOfForest/extentOfForestValidatorState.js

```diff
@@ -48,6 +48,19 @@ export const otherWoodedLandValidator = datum => state => {
   return areasNotExceedingTotalLandArea && hasValue
 }
 
+// ==== Common validator
+
+export const lessThanOrEqualToForestValidator = (year, value) => state => {
+  const forest = ExtentOfForestState.getForestByYear(year)(state)
+
+  if (R.isNil(value) || R.isNil(forest)) {
+    return true
+  }
+  const tolerance = -1
```

Comment:
can we set this to default value for greaterThan?

Reply:
I noticed that. on one side yes, on the other making it explicitly serves as doc. 
Plus, this logic is used several times, i realized that in the end. I would suggest to add this method to numberUtils and in a separate issue. what do you think?

Reply:
Agree